root_utils/event_dump.py

```diff
@@ -101,6 +109,8 @@ def dump_file(infile, outfile):
         event_id[ev] = ev
         root_file[ev] = infile
 
+        print(ev)
```

Comment:
oh forgot to remove this printout?

Reply:
[removed](https://github.com/WatChMaL/DataTools/commit/4ea8d925fdce19eaa06f807d418a87fb67a30133)